tools/generators/migrate-converged-pkg/index.spec.ts

```diff
@@ -1,3 +1,4 @@
+import * as Enquirer from 'enquirer';
```

Comment:
why is this `PascalCase` ? `enquirer` exports a class with static methods (not the best way for tree-shakeability, it is what it is)